### PR TITLE
Fix Configuration Management Link

### PR DESCRIPTION
**Pull request by psav**
Purpose or Intent
=================

Fixing link name

### CODE REVIEW

File: cfme/infrastructure/config_management.py [hunk 1]
```diff
@@ -442,7 +442,10 @@ class MgrAll(CFMENavigateStep):
 
     def step(self):
         from cfme.web_ui.menu import nav
-        nav._nav_to_fn('Configuration', 'Configuration Management')(None)
+        if self.obj.appliance.version > '5.7.0.8':
+            nav._nav_to_fn('Configuration', 'Configuration')(None)
```
Comment (patchkez) [inline cfme/infrastructure/config_management.py]: @psav, If I am not mistaken, this line should be changed to:

`
nav._nav_to_fn('Configuration', 'Management')(None)
`

screenshot-20161102 112717

I did not test the code, but it looks good to me.

Comment (psav) [inline cfme/infrastructure/config_management.py]: doh!
ima idiot :)

**Comment by dajoRH** [comment]
Lint report for commit b1e519c6e19b6bf95f38e54ae26d8a5b63aa466d:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by patchkez** [comment]
@psav  - please merge.

**Pull request merged**